pkg/controller/plan/migration.go

```diff
@@ -1726,6 +1733,49 @@ func (r *Migration) setPopulatorPodsWithLabels(vm *plan.VMStatus, migrationID st
 	}
 }
 
+func (r *Migration) getVirtV2VDiskConfig(vm *plan.VMStatus, step *plan.Step) (err error) {
```

Comment:
maybe there is a better placement to this function, the update of the migration part might fit kubevirt.go